pkg/zng/docs/spec.md

```diff
@@ -78,7 +78,7 @@ For example,
 9:no, there isn't.
 4:3
 ```
-where type code 4 represents an integer.  This encoding represents the string of
+where type code 4 represents an integer.  This encoding represents the sequence of
```

Comment:
I figured "string" was ambiguous here since it's also a reference to one of our actual data types.